test/Interpreter/SDK/cf_without_foundation.swift

```diff
@@ -0,0 +1,61 @@
+// RUN: %target-run-simple-swift | %FileCheck %s
+
+// REQUIRES: executable_test
+// REQUIRES: objc_interop
```

Comment:
Technically only requires a platform where CF is part of the SDK, doesn't it?

What do you think about moving/copying this test in to Foundation and running it on all platforms?